templates/etc/fastd/fastd.conf.erb

```diff
@@ -12,7 +12,20 @@ hide mac addresses yes;
 include "secret.conf";
 mtu <%= @mesh_mtu %>; # 1492 - IPv{4,6} Header - fastd Header...
 status socket "/var/run/fastd-status.<%= @mesh_code %>.sock";
-include peers from "peers";
+peer group "nodes" {
+  include peers from "peers";
+<% if @peer_limit %>
```

Comment:
does this work? isn't peer_limit always defined, as it is a parameter of the fastd module? maybe you should make peer_limit a parameter with a default value of e.g. -1 or something, so that this value represents disabling the peer limit altogether